install

```diff
@@ -20,7 +20,7 @@ function run() {
     echo "Installing with no inventory git url; set later in $CONFIG_FILE"
   fi
 
-  if [[ -e /var/lib/aviary ]]; then 
+  if [[ -e /var/lib/aviary ]]; then
```

Comment:
This should be updated to $INSTALL_PATH.

Reply:
woops, yep, u right.

Reply:
K. got it.